examples/2d/texture_atlas.rs

```diff
@@ -49,10 +56,184 @@ fn setup(
+            translation: Vec3::new(-250.0, -130.0, 0.0),
+            scale: Vec3::splat(0.8),
+            ..default()
+        },
+        ..default()
+    });
+
+    // draw padded texture atlas
+    commands.spawn(SpriteBundle {
+        texture: texture_atlas_linear_padded.texture,
+        transform: Transform {
+            translation: Vec3::new(250.0, -130.0, 0.0),
+            scale: Vec3::splat(0.8),
+            ..default()
+        },
+        ..default()
+    });
+
+    // draw sprites from texture atlases
+
+    // get handle to a sprite to render
+    let vendor_handle: Handle<Image> = asset_server
+        .get_handle("textures/rpg/chars/vendor/generic-rpg-vendor.png")
+        .unwrap();
+
+    // get index of the sprite in the texture atlas, this is used to render the sprite
+    // the index is the same for all the texture atlases, since they are created from the same folder
+    let vendor_index = texture_atlas_linear
+        .get_texture_index(&vendor_handle)
+        .unwrap();
+
+    // linear, no padding
+    create_sprite_from_atlas(
+        &mut commands,
+        (-350.0, 170.0, 0.0),
+        vendor_index,
+        atlas_linear_handle,
+    );
+
+    // nearest, no padding
+    create_sprite_from_atlas(
+        &mut commands,
+        (-150.0, 170.0, 0.0),
+        vendor_index,
+        atlas_nearest_handle,
+    );
+
+    // linear, padding
+    create_sprite_from_atlas(
+        &mut commands,
+        (150.0, 170.0, 0.0),
+        vendor_index,
+        atlas_linear_padded_handle,
+    );
+
+    // nearest, padding
+    create_sprite_from_atlas(
+        &mut commands,
+        (350.0, 170.0, 0.0),
+        vendor_index,
+        atlas_nearest_padded_handle,
+    );
+
+    // labels to indicate the different settings
+    let font = asset_server.load("fonts/FiraSans-Bold.ttf");
+    let text_style = TextStyle {
+        font: font.clone(),
+        font_size: 50.0,
+        color: Color::WHITE,
+    };
+
+    // No padding
+    create_label(
+        &mut commands,
+        (-250.0, 330.0, 0.0),
+        "No padding",
+        text_style.clone(),
+    );
+
+    // Padding
+    create_label(&mut commands, (250.0, 330.0, 0.0), "Padding", text_style);
+
+    let text_style_smaller = TextStyle {
+        font,
+        font_size: 30.0,
+        color: Color::WHITE,
+    };
+
+    // Linear, left
+    create_label(
+        &mut commands,
+        (-350.0, 280.0, 0.0),
+        "Linear",
+        text_style_smaller.clone(),
+    );
+
+    // Nearest, left
+    create_label(
+        &mut commands,
+        (-150.0, 280.0, 0.0),
+        "Nearest",
+        text_style_smaller.clone(),
+    );
+
+    // Linear, right
+    create_label(
+        &mut commands,
+        (150.0, 280.0, 0.0),
+        "Linear",
+        text_style_smaller.clone(),
+    );
+
+    // Nearest, right
+    create_label(
+        &mut commands,
+        (350.0, 280.0, 0.0),
+        "Nearest",
+        text_style_smaller,
+    );
+}
```

Comment:
The repeated `create_label` calls could be refactored using a closure or loop maybe. It's not blocking though.